modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -117,6 +117,15 @@
         end
       end
 
+      it 'increment counters for statsd' do
+        with_okta_user(scopes) do |auth_header|
+          VCR.use_cassette('evss/disability_compensation_form/form_526_invalid_validation') do
+            expect(StatsD).to receive(:increment).exactly(6).times
```

Comment:
This test seems fragile and would fail if someone added another metric and only 3 of the `StatsD.increment` calls are part of the newly added metrics.  Is there anything we can do to make the test more specific?  (it's possible it's too big of a hassle)

Reply:
I'll change this to `at_least(:once)` seems way better thanks

Reply:
resolved here: https://github.com/department-of-veterans-affairs/vets-api/pull/3208/commits/914ebffce8c62366c3bc6d6cc8b2571761834bdc